lib/components/component.dart

```diff
@@ -16,6 +17,10 @@ abstract class Component {
   /// HUD objects ignore the [BaseGame.camera] when rendered (so their position coordinates are considered relative to the device screen).
   bool isHud = false;
 
+  /// Whether this component is currently mounted on a game or not
```

Comment:
I would move this dartdoc to the getter. Otherwise IDE's wont display the dartdoc comment considering this is a private property :-)

Reply:
Done!